globus_sdk/auth/token_response.py

```diff
@@ -56,11 +60,19 @@ def by_resource_server(self):
         return self._by_resource_server
 
     @property
+    def expires_at(self):
+        """
+        A POSIX timestamp representation of the expiration
+        time for the top-level token in the response.
+        """
+        return self['expires_at']
```

Comment:
This doesn't seem right. Lookups on `self` hit `self.data`, the parsed API response (inherited from `GlobusResponse`), which won't have an `expires_at` key in it.

What you really want is, I think, `self.by_resource_server[self.resource_server]['expires_at_seconds']`, which is ugly as sin, and maybe suggests that we shouldn't bother or need to restructure this.

I was iffy when I added these convenience properties for the top-level token, maybe we should just drop them?